kpops/component_handlers/helm_wrapper/model.py

```diff
@@ -87,6 +87,11 @@ class YamlReader:
     def __iter__(self) -> Iterator[str]:
         # discard all output before template documents
         index = self.content.index("---")
-        self.content = self.content[index:-1]
+        notes = "\n\nNOTES:\n"
+        if notes in self.content:
+            rindex = self.content.index(notes)
```

Comment:
perhaps we should rename the variables `index` and `rindex` as the name is a bit misleading now. since `rindex` refers to the last occurrence
how about `start` & `end`?

Reply:
agreed